brainscore/submission/evaluation.py

```diff
@@ -24,47 +22,64 @@
                        if benchmark not in ['dicarlo.Kar2019-ost', 'fei-fei.Deng2009-top1']]
```

Comment:
we should figure out a way soon to run these efficiently. Maybe execution in parallel would help so that we don't have to wait for every single benchmark to finish?

Reply:
I could run models in parallel, but I wouldn't do the same with benchmarks. After all, they don't take too long once all data is loaded.

Reply:
ImageNet takes a long time IIRC, that's mostly why we're excluding it (and it requires GPU). The number of benchmarks will also increase more and more. (no need to address this in this PR but maybe something we should discuss)